app/scripts/modules/dcos/help/dcos.help.ts

```diff
@@ -0,0 +1,38 @@
+import {module} from 'angular';
+import {HELP_CONTENTS_REGISTRY, HelpContentsRegistry} from '@spinnaker/core';
```

Comment:
add spaces between brackets and contents, e.g. `{ module }` instead of `{module}`.

(we are not very consistent throughout the codebase but, moving forward, trying to enforce this convention)